adapters/adxcg/adxcg.go

```diff
@@ -0,0 +1,130 @@
+package adxcg
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb/v14/openrtb2"
+	//"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// Builder builds a new instance of the Adxcg adapter for the given bidder with the given config.
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter) (adapters.Bidder, error) {
+	bidder := &AdxcgAdapter{
+		endpoint: config.Endpoint,
+	}
+	return bidder, nil
+}
+
+// MediaOptiAdapter implements the Bidder interface.
+type AdxcgAdapter struct {
+	endpoint string
+}
+
+// MakeRequests prepares the HTTP requests which should be made to fetch bids.
+func (adapter *AdxcgAdapter) MakeRequests(
+	openRTBRequest *openrtb2.BidRequest,
+	reqInfo *adapters.ExtraRequestInfo,
+) (
+	requestsToBidder []*adapters.RequestData,
+	errs []error,
+) {
+	openRTBRequestJSON, err := json.Marshal(openRTBRequest)
+	if err != nil {
+		errs = append(errs, err)
+		return nil, errs
+	}
+
+	headers := http.Header{}
+	headers.Add("Content-Type", "application/json;charset=utf-8")
+	requestToBidder := &adapters.RequestData{
+		Method:  "POST",
+		Uri:     adapter.endpoint,
+		Body:    openRTBRequestJSON,
+		Headers: headers,
+	}
+	requestsToBidder = append(requestsToBidder, requestToBidder)
+
+	return requestsToBidder, errs
+}
+
+const unexpectedStatusCodeFormat = "" +
+	"Unexpected status code: %d. Run with request.debug = 1 for more info"
```

Comment:
I'm curious why you include the ` "" +`?

Reply:
cleaned up